lib/t/feeds.ex

```diff
@@ -625,4 +629,149 @@ defmodule T.Feeds do
     |> where([s], s.inserted_at < fragment("now() - ? * interval '1 day'", ^ttl_days))
     |> Repo.delete_all()
   end
+
+  # newbies and stuff
+
+  @doc """
+  Starts a "Since Live" event for newbies:
+
+    - sends a message to the admin tg bot
+    - broadcasts a push notification to all participants
+    - broadcasts a `:start` event to all newbies' feed channels
+
+  This function shouldn't be called directly, but rather scheduled with Oban in crontab.
+  """
+  @spec newbies_start_live(DateTime.t()) :: :ok
+  def newbies_start_live(reference \\ DateTime.utc_now()) do
+    newbies = newbies_list_today_participants(reference)
+
+    m = "LIVE (newbie's special edition) started: newbies count #{length(newbies)}"
+    Logger.warn(m)
+    Bot.async_post_message(m)
+
+    Enum.each(newbies, fn user_id ->
+      feed_channel_topic = "feed:" <> user_id
+      broadcast(feed_channel_topic, {__MODULE__, [:mode_change, :start]})
+    end)
+
+    newbies
+    |> T.Accounts.list_apns_devices()
+    |> DispatchJob.schedule_apns("newbie_live_mode_started", _data = %{})
+
+    :ok
+  end
+
+  @doc """
+  Ends a "Since Live" event for newbies:
+
+    - sends a message to the admin tg bot
+    - broadcasts a `:end` event to all `:mode_change` subscribers
+    - clears live tables
+
+  This function shouldn't be called directly, but rather scheduled with Oban in crontab.
+  """
+  @spec newbies_end_live :: :ok
+  def newbies_end_live do
+    m = "LIVE (newbie's special edition) ended"
+    Logger.warn(m)
+    Bot.async_post_message(m)
+
+    # we don't need to send `end` event only to newbies,
```

Comment:
~~would break current frontend: it will change the mode for non-newbies
https://github.com/getsince/test5/blob/rails-fork/Since/Feeds/Live%20Feed/LiveFeedChannel.swift#L60
so better send this event only to newbies

(though I recognize that this frontend "feature" should be fixed)~~